frontend_vue/src/components/base/BaseCodeSnippet.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div>
+    <label
+      v-if="label"
+      class="inline-block mb-8 text-grey-500"
+      :for="label"
+      >{{ label }}</label
+    >
+
+    <div class="relative border rounded-lg border-grey-100">
+      <div class="absolute top-[.8rem] right-[1rem] z-10 flex gap-8">
+        <BaseRefreshButton
+          v-if="hasRefresh"
+          @refresh-token="handleRefreshToken"
```

Comment:
what is this for exactly?

Reply:
We have this functionality in some places
<img width="502" alt="Screenshot 2024-03-27 at 15 08 02" src="https://github.com/thinkst/canarytokens/assets/126554007/6efe6e27-1701-4793-9d06-5dc66c93139e">

( it looks like it's broken on prod 👀 is it just me? )

Reply:
oh i see... yes it looks broken eish. ok we can fix it in this version heh